release_dashboard/lib/widgets/engine_cherrypicks_substeps.dart

```diff
@@ -62,39 +68,80 @@ class ConductorSubstepsState extends State<EngineCherrypicksSubsteps> {
 
   @override
   Widget build(BuildContext context) {
+    List<String> engineCherrypicksInConflict = <String>[];
+    if (context.watch<StatusState>().releaseStatus != null &&
+        context.watch<StatusState>().releaseStatus?['Engine Cherrypicks'] != null) {
+      for (Map<String, String> engineCherrypick
+          in context.watch<StatusState>().releaseStatus?['Engine Cherrypicks'] as List<Map<String, String>>) {
+        if (engineCherrypick['state'] ==
+            EngineCherrypicksSubsteps.cherrypickStates[pb.CherrypickState.PENDING_WITH_CONFLICT]) {
+          engineCherrypicksInConflict.add(engineCherrypick['trunkRevision']!);
+        }
+      }
+    }
+
     return Column(
       children: <Widget>[
         CheckboxAsSubstep(
-          substepName: EngineCherrypicksSubsteps.substepTitles[EngineCherrypicksSubstep.substep1]!,
-          subtitle: SelectableText(EngineCherrypicksSubsteps.substepSubtitles[EngineCherrypicksSubstep.substep1]!),
-          isChecked: _isEachSubstepChecked[EngineCherrypicksSubstep.substep1]!,
-          clickCallback: () {
-            substepPressed(EngineCherrypicksSubstep.substep1);
-          },
-        ),
-        CheckboxAsSubstep(
-          substepName: EngineCherrypicksSubsteps.substepTitles[EngineCherrypicksSubstep.substep2]!,
-          subtitle: SelectableText(EngineCherrypicksSubsteps.substepSubtitles[EngineCherrypicksSubstep.substep2]!),
-          isChecked: _isEachSubstepChecked[EngineCherrypicksSubstep.substep2]!,
+          substepName: EngineCherrypicksSubsteps.substepTitles[EngineCherrypicksSubstep.verifyRelease]!,
+          subtitle: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              SelectableText(
+                  'Verify if the release number: ${context.watch<StatusState>().releaseStatus?['Release Version']}'
```

Comment:
Prefer assigning to this a variable at the start of your build function. It's easier to debug if there's a dedicated line for pulling this information.

Reply:
Are you referring to bring `context.watch<StatusState>` to the start of the build function? `context.watch<StatusState>` is being called several times in this widget.

Reply:
Yes. It might just work if you assign the StatusState to a variable